src/main/java/cc/hyperium/mixinsimp/renderer/HyperiumRender.java

```diff
@@ -6,7 +6,7 @@
 import cc.hyperium.event.RenderNameTagEvent;
 import cc.hyperium.mixins.renderer.IMixinRender;
 import cc.hyperium.utils.ChatColor;
-import cc.hyperium.utils.StaffUtils;
+import cc.hyperium.utils.staff.StaffUtils;
```

Comment:
Was this a Refactor or mistake?

Reply:
moved the staffutils class to that package since the other is there aswell